pkg/api/types.go

```diff
@@ -86,6 +86,14 @@ const (
 
 // NetworkProfile contains configuration for OpenShift networking.
 type NetworkProfile struct {
+	// VnetID is expected to be empty or match
```

Comment:
Same as above.

Reply:
Moved VnetID field below VnetCIDR, edited the comment to describe contents of both fields.